androidapp/Crisislab/app/src/main/kotlin/com/example/crisislab/NotificationHandler.kt

```diff
@@ -13,35 +13,39 @@ import com.example.crisislab.NotificationModule
 
 @RequiresApi(Build.VERSION_CODES.TIRAMISU)
 class NotificationHandler (
+    // Notification manager for displaying notifications
     private val notificationManager: NotificationManagerCompat,
-
+    // Application context
     private val context: Context
 ) : Service() {
-    var isServiceRunning = false;
-
+    var isServiceRunning = false
 
+    // Binding the service to a client (not used)
     override fun onBind(intent: Intent?): IBinder? {
         return null!!
     }
 
+    // Handle service start commands
```

Comment:
why

Reply:
Also I just noticed `return null!!`... that will just immediately throw an exception cos you're trying to assert that null isn't null

Reply:
This whole file is an absurd mess. In fairness It's not my code so I'll get Max to tidy it up.